sriovnet.go

```diff
@@ -407,6 +409,24 @@ func GetVfNetdevName(handle *PfNetdevHandle, vf *VfObj) string {
 	return vfNetdevNameFromParent(handle.PfNetdevName, vf.Index)
 }
 
+// Finds the VF index of a port, if applicable
+func GetVfIndexFromNetDevice(name string) (int, error) {
```

Comment:
This method is only relevant for switchdev right ?

switchdev related functionality should go to sriovnet_switchdev.go

also you should check if device is in switchdev mode first, similarly to how its done here:
https://github.com/Mellanox/sriovnet/blob/e360783a88a929a3100628cb7ce27ed029c276f9/sriovnet_switchdev.go#L129

in addition need to add unit test for added functionality